lib/moon/design/tag.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Moon.Design.Tag do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+
+  prop(size, :string, default: "xs", values: ~w(2xs xs))
+  prop(class, :css_class)
+  prop(is_uppercase, :boolean, default: true)
+
+  slot(default)
+  slot(left_icon)
+  slot(right_icon)
+
+  def render(assigns) do
+    ~F"""
+    <div class={
+      merge([
+        "flex items-center rounded-moon-i-xs select-none text-gohan bg-bulma",
+        set_letter_spacing(@size),
```

Comment:
i'm not sure that having this lot of functions is a good idea? isn't it more transparent to put as clear conditional class list depending on `@size` attriobute?

Reply:
Valid point. Functions now replaced.